bundle/manifests/clusterlogging.clusterserviceversion.yaml

```diff
@@ -422,7 +422,7 @@ spec:
                 - name: RELATED_IMAGE_LOG_FILE_METRIC_EXPORTER
                   value: quay.io/openshift-logging/log-file-metric-exporter:latest
                 - name: RELATED_IMAGE_LOGGING_CONSOLE_PLUGIN
-                  value: quay.io/openshift-logging/logging-view-plugin:latest
+                  value: quay.io/openshift-logging/logging-view-plugin:5.6
```

Comment:
This file has a reference to `logging-view-plugin:latest` in [L546](https://github.com/openshift/cluster-logging-operator/pull/1949/files#diff-96ec3d225361a6dcf1232b3905c372888ea85a0b0fe5598e97df72ccd7c53dadR546) as well that needs to be updated. It is generated from the `manager.yaml`.